src/forms.ts

```diff
@@ -76,5 +86,5 @@ export function hasErrors(body: SubmissionBody): body is ErrorBody {
 
 export interface SubmissionResponse {
   body: SubmissionBody;
-  response: Response;
+  response: Response | null;
```

Comment:
Can this be 

`response?: Response`

?

Reply:
I'll check

Reply:
https://github.com/formspree/formspree-core/pull/7#discussion_r906732550